client/src/app/pages/importer-list/importer-list.tsx

```diff
@@ -540,8 +540,8 @@ export const ImporterExpandedArea: React.FC<ImporterExpandedAreaProps> = ({
     isSortEnabled: true,
     sortableColumns: ["startDate", "endDate"],
     getSortValues: (report) => ({
-      startDate: dayjs(report.report.startDate).millisecond(),
-      endDate: dayjs(report.report.startDate).millisecond(),
+      startDate: dayjs(report.report.startDate).valueOf(),
+      endDate: dayjs(report.report.startDate).valueOf(),
```

Comment:
```suggestion
      endDate: dayjs(report.report.endDate).valueOf(),
```

Reply:
This is my own mistake. I used `startDate` in both lines. So it would be a good chance to correct my mistake